man/fwrite.Rd

```diff
@@ -62,6 +63,7 @@ fwrite(x, file = "", append = FALSE, quote = "auto",
   \item{bom}{If \code{TRUE} a BOM (Byte Order Mark) sequence (EF BB BF) is added at the beginning of the file; format 'UTF-8 with BOM'.}
   \item{verbose}{Be chatty and report timings?}
   \item{encoding}{ The encoding of the strings written to the CSV file. Default is \code{""}, which means writing raw bytes without considering the encoding. Other possible options are \code{"UTF-8"} and \code{"native"}. }
+  \item{forceDecimal}{ Should decimal points be forced for whole numbers in numeric columns? When \code{TRUE}, whole numbers like \code{c(1.0, 2.0, 3.0)} will be written as \code{1., 2., 3.} instead of \code{1, 2, 3}. Default is \code{FALSE}. }
```

Comment:
I guess the description needs to avoid assuming `dec='.'`. Also, `1., 2., 3.` is not valid R code, so we need `\samp`.

```suggestion
  \item{forceDecimal}{ Should decimal points be forced for whole numbers in numeric columns? When \code{FALSE}, the default, whole numbers like \code{c(1.0, 2.0, 3.0)} will be written as \samp{1, 2, 3} i.e., dropping \code{dec}. }
```